hack/build_docker.sh

```diff
@@ -12,31 +12,38 @@ case "$TARGET" in
 	"enduro")
 		IMAGE_NAME="enduro"
 		TARGET="enduro"
+		FOLDER="."
 		;;
 	"enduro-a3m-worker")
 		IMAGE_NAME="enduro-a3m-worker"
 		TARGET="enduro-a3m-worker"
+		FOLDER="."
+		;;
+        "enduro-dashboard")
+		IMAGE_NAME="enduro-dashboard"
+		TARGET="enduro-dashboard"
+		FOLDER="dashboard"
 		;;
 	*)
-		echo "Accepted values: enduro, enduro-a3m-worker.";
+		echo "Accepted values: enduro, enduro-a3m-worker, enduro-dashboard.";
 		exit 1;
 		;;
 esac
 
 eval $(./hack/build_dist.sh shellvars)
 
-DEFAULT_IMAGE_NAME="${IMAGE_NAME}:${VERSION_SHORT}"
+DEFAULT_IMAGE_NAME="${IMAGE_NAME}:${2:-${VERSION_SHORT}}"
 TILT_EXPECTED_REF=${EXPECTED_REF:-}
 IMAGE_NAME="${TILT_EXPECTED_REF:-$DEFAULT_IMAGE_NAME}"
 BUILD_OPTS="${BUILD_OPTS:-}"
 
 env DOCKER_BUILDKIT=1 docker build \
 	-t "$IMAGE_NAME" \
-	-f "Dockerfile" \
+	-f "$FOLDER/Dockerfile" \
 	--build-arg="TARGET=$TARGET" \
 	--build-arg="VERSION_PATH=$VERSION_PATH" \
 	--build-arg="VERSION_LONG=$VERSION_LONG" \
 	--build-arg="VERSION_SHORT=$VERSION_SHORT" \
 	--build-arg="VERSION_GIT_HASH=$VERSION_GIT_HASH" \
 	$BUILD_OPTS \
-		.
+        $FOLDER
```

Comment:
@scollazo here you're using spaces instead of tabs, but I'll merge and fix later in a following PR.